ci/nightly.sh

```diff
@@ -22,7 +22,9 @@ build_nightly() {
     cd ${NEOVIM_DIR}
     if [ "${CI_OS}" = osx ] ; then
       # This CMAKE_EXTRA_FLAGS is required for relocating the macOS libs.
-      make CMAKE_BUILD_TYPE=Release CMAKE_EXTRA_FLAGS="-DENABLE_JEMALLOC=OFF -DCMAKE_INSTALL_PREFIX:PATH="
+      make CMAKE_BUILD_TYPE=Release \
+           CMAKE_EXTRA_FLAGS="-DENABLE_JEMALLOC=OFF -DCMAKE_INSTALL_PREFIX:PATH=" \
+           CMAKE_OSX_DEPLOYMENT_TARGET="10.13"
```

Comment:
are you sure this works? shouldn't it be in CMAKE_EXTRA_FLAGS like this:

     CMAKE_EXTRA_FLAGS="... -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 ..."